CMakeLists.txt

```diff
@@ -38,7 +38,10 @@ option( VERBOSE "Show information about CMake build configuration." )
 
 # Enable C++11 builds
 # TODO: With CMake > 3.1, can use target_compile_features() to do this automatically
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
+#set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
+set(CMAKE_CXX_STANDARD 11)
```

Comment:
This needs CMake >= 3.1, so we'd need to update the requirement here (and in the debian/config in the master-debian branch)